pkgs/by-name/in/inflow/package.nix

```
@@ -0,0 +1,76 @@
+{ lib, stdenv, fetchFromGitHub, runCommand, inflow, diffutils }:
+
+stdenv.mkDerivation rec {
+  pname = "inflow";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "stephen-huan";
+    repo = "inflow";
+    rev = "v${version}";
+    sha256 = "sha256-xKUqkrPwITai8g6U1NiNieAip/AzISgFfFtvR30hLNk=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+
+    $CXX -Wall -Wpedantic -Wextra -O3 -o inflow inflow.cpp
```

```suggestion
    $CXX -Wall -Wpedantic -Wextra -o inflow inflow.cpp
```

I recommend not introducing additional optimization parameters.

---

If you don't mind, what's the reasoning (e.g. are there reproducibility issues)? 

I ask because there seem to be [many](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs+-O3&type=code) packages that use `-O3` already.